.buildkite/autogenerate_pipeline.py

```diff
@@ -122,6 +129,12 @@ def _add_docker_config(self, cfg):
             for key, val in cfg.items():
                 target[key] = val
 
+    def _set_timeout(self, timeout):
```

Comment:
I would keep only the env variable setting.

Reply:
Removed the method.